cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc

```diff
@@ -75,8 +87,9 @@ In summary, `TEXT` indexes support the following predicates:
 When multiple indexes are available and able to solve a predicate, there is an order defined that decides which index to use.
 It is defined as such:
 
-* `TEXT` indexes are preferred over `BTREE` indexes for `CONTAINS` and `ENDS WITH`.
-* `BTREE` indexes are preferred over `TEXT` indexes in all other cases.
+* `TEXT` indexes are used over `RANGE` and `POINT` indexes for `CONTAINS` and `ENDS WITH`.
+* `POINT` indexes are used over `RANGE` and `TEXT` indexes for distance and within bounding box.
```

Comment:
```suggestion
* `POINT` indexes are used over `RANGE` and `TEXT` indexes for distance and within a bounding box.
```
Alternatively: 
```* `POINT` indexes are used over `RANGE` and `TEXT` indexes for distance and within bounding boxes.```

Reply:
I am a bit unsure about this one, since the query is actually called withinBBox, but i guess that the adding "a" is more grammatically correct.  I don't think that the alternative is correct though since that implies that it is used for multiple bounding boxes.